docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
+
+## General Environment Setup
+
+### Declare Variables
+
+Some global variables used throughout the overlay setup.
+
+- `ZITI_HOME` is the directory where your network files will be stored, the directory will be created if it does not exist already.
+- `ZITI_BIN_DIR` is the directory where the ziti binary will be downloaded and extracted
+- `ZITI_PWD` is the password that will be used when logging into the controller as the default user (admin).
+- `ZITI_HOSTNAME` is used throughout the setup for naming files, network elements, etc.
+
+```
+export ZITI_HOME="${HOME}/.ziti"
+export ZITI_BIN_DIR="${ZITI_HOME}/ziti_bin"
+export ZITI_PWD=admin2
+export ZITI_HOSTNAME=$(hostname -s)
+```
+
+### Create Environment
+
+Create a directory where you will store your network files.
+
+```
+mkdir -p "${ZITI_HOME}"
+```
+
+### Obtain Ziti Binary
+
+The ziti binary is required to setup the network. The exact URL will differ depending on your operating system and architecture. Visit the [releases](https://github.com/openziti/ziti/releases) page to get the appropriate URL.
+
+```
+mkdir -p "${ZITI_BIN_DIR}"
+curl -L <release-url> -o "${ZITI_BIN_DIR}/ziti-bin.gz"
+tar -xf "${ZITI_BIN_DIR}/ziti-bin.gz" --directory "${ZITI_BIN_DIR}"
+chmod +x "${ZITI_BIN_DIR}/ziti"
+```
+
+## Create PKI
+
+You may use an existing PKI if you prefer, however, as part of the `expressInstall` a PKI is generated for you. The following will run through the process of generating your own PKI.
```

Comment:
> You may use an existing PKI if you prefer

There's no option to do this? This page is explaining the steps that the quickstart DOES. it most definitely makes a new pki every time so it's not a preference, right?